package.json

```
@@ -26,7 +26,12 @@
   "scripts": {
     "watch": "cds watch",
     "start": "cds-serve",
-    "test": "npx jest --silent"
+    "test": "npx jest --silent",
+    "add-change-tracking": "npm i @cap-js/change-tracking && cp xmpls/change-tracking.cds ./srv",
+    "add-telemetry": "npm i @cap-js/telemetry",
+    "add-attachments": "npm i @cap-js/attachments && cp xmpls/attachments.cds ./srv",
+    "add-notifications": "npm i @cap-js/notifications && cp xmpls/alert-notifications.js ./srv && cp xmpls/notification-types.json ./srv",
```

Very minor cosmetics only:
```suggestion
    "add-change-tracking": "npm add @cap-js/change-tracking && cp xmpls/change-tracking.cds ./srv",
    "add-telemetry": "npm add @cap-js/telemetry",
    "add-attachments": "npm add @cap-js/attachments && cp xmpls/attachments.cds ./srv",
    "add-notifications": "npm add @cap-js/notifications && cp xmpls/alert-notifications.js ./srv && cp xmpls/notification-types.json ./srv",
```

---

thank you, that's fixed now!